packages/@aws-cdk-testing/framework-integ/test/aws-cognito/test/integ.user-pool-group.ts

```diff
@@ -0,0 +1,37 @@
+import { App, RemovalPolicy, Stack } from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import { UserPool, UserPoolGroup } from 'aws-cdk-lib/aws-cognito';
+import { Role, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+    const userPool = new UserPool(this, 'UserPool', {
+      removalPolicy: RemovalPolicy.DESTROY,
+    });
+
+    const role = new Role(this, 'Role', {
+      assumedBy: new ServicePrincipal('cognito-idp.amazonaws.com'),
+    });
+
+    new UserPoolGroup(this, 'UserPoolGroup', {
+      userPool: userPool,
+      groupName: 'test-group',
+      description: 'My user pool group',
+      precedence: 0,
```

Comment:
Maybe overkill, but can we also add an integ test to confirm that a `UserPoolGroup` without `precedence` passed in deploys?

Reply:
Thank you. I modified `AnotherUserPoolGroup` to deploy without `precedence`.